docs/index.html

```diff
@@ -466,6 +466,75 @@ <h2>Key Management</h2>
         </p>
       </section>
 
+      <section class="normative">
+        <h2>Selective Disclosure from Credentials</h2>
+        <p class="issue">
+          All current approaches to selective discloser are active works in progress with
+          notable security or other considerations.  These are actively being tracked in
+          <a href="https://github.com/w3c-ccg/traceability-interop/issues/39">Issue #39</a>. 
+          Selective Disclosure using any mechansim SHOULD NOT be used for production data 
+          at this time until further testing and security analysis has been performed.
+        </p>
+        <p>
+          Selective disclosure of certain values from signed credentials is a highly desireable
+          feature that is active being worked towards.  At the moment, two general approaches 
+          exist for this particular feature:
+          <ul>
+            <li>BBS+</li>
+            <li>Merkle Tree Proofs</li>
+          </ul>
+        </p>
+        <section class="normative">
+          <h2>BBS+</h2>
+          <p>
+            BBS+ approaches to signatures for use with selective disclosure rely on pairing
+            friendly curves, specifically BLS12-381 in the current specifications.
+          </p>
+          <p class="issue">
+            <a href="https://w3c-ccg.github.io/ldp-bbs2020/">BBS+ Signatures 2020</a> 
+            MUST NOT be used due to issues related to one or more blank nodes as well
+            as sensitivity around the ordering of a signed message.  See issue
+            <a href="https://github.com/w3c-ccg/ldp-bbs2020/issues/62">#62</a> for more
+            details.
+          </p>
+          <p>
+            The <a href="https://github.com/decentralized-identity/bbs-signature">
+              BBS+ Signature Scheme
+            </a> specification is consolidating work towards an updated version of 
+            BBS+ signatures that aim to resolve prior issues.
+          </p>
+        </section>
+        <section class="normative">
+          <h2>Merkle Tree Proofs</h2>
+          <p>
+            <a href="https://computersciencewiki.org/index.php/Merkle_proof">Merkle Tree</a> 
+            based proofs for selective disclosure attempt to resolve
+            concerns around requirements for pairing friendly curves leveraged in
+            BBS+ and other approaches to selective disclosure by not mandating the
+            hashing and signature suite mechansim so that those algorithms may be
+            selected to meet the needs of a particular operating envrionment.
+          </p>
+          <p>
+            At the current time, merkle trees have test implementations using both
```

Comment:
explicitly state here that it must not be used in production as its test implementation ( similar to the note above in BBS+)
otherwise I love this!

Reply:
resolved per commit below